guide/kh/index.md

```diff
@@ -0,0 +1,5 @@
+---
+section: guide
+lang: KH
+title: Khmer
+---
```

Comment:
OK, got it. The event looks great! 
I think it will be better if I will close this pull request, and you will submit a new one with the right language code. :-)
If you want I can help you on Skype as well before the event. Drop me an email - mor.rubinstein@okfn.org